src/rust/engine/Cargo.toml

```diff
@@ -116,11 +116,13 @@ members = [
   "cache",
   "client",
   "concrete_time",
+  "dep_inference",
   "fs",
   "fs/brfs",
   "fs/fs_util",
   "fs/store",
   "graph",
+  "grpc_util",
```

Comment:
Potentially this and `photos` were intentionally omitted? (A quick skim of the rather long discussion on #11307 that introduced them didn't make this obvious.)